man/systemd.netdev.xml

```diff
@@ -183,6 +183,8 @@
           <row><entry><varname>ifb</varname></entry>
           <entry> The Intermediate Functional Block (ifb) pseudo network interface acts as a QoS concentrator for multiple different sources of traffic.</entry></row>
 
+          <row><entry><varname>bareudp</varname></entry>
+          <entry> The Bare UDP tunnel module provides a generic L3 encapsulation support for tunnelling different L3 protocols like MPLS, IP etc inside an UDP tunnel.</entry></row>
```

Comment:
"The Bare UDP tunnel module provides a" → "Bare UDP tunnels provide"

inside an → inside of an

etc → etc.